docs/source/data/errors.mdx

```diff
@@ -522,19 +541,14 @@ const setHttpPlugin = {
         if (response?.errors?.[0]?.message === 'teapot') {
           response.http.status = 418;
```

Comment:
i changed this to statusCode but changed it back to status today since that seemed like an unnecessary annoyance :)